src/transformers/deepspeed.py

```diff
@@ -111,6 +111,24 @@ def get_value(self, ds_key_long, default=None):
             return default
         return config.get(ds_key, default)
 
+    def del_config_sub_tree(self, ds_key_long, must_exist=False):
+        config = self.config
+
+        # find the config node of interest if it exists
+        nodes = ds_key_long.split(".")
+        for node in nodes:
+            parent_config = config
+            config = config.get(node)
+            if config is None:
+                if must_exist:
+                    raise ValueError(f"Can't find {ds_key_long} entry in the config")
```

Comment:
Might be worth printing the config as well?
```suggestion
                    raise ValueError(f"Can't find {ds_key_long} entry in the config {config}")
```

Reply:
that won't work, but I will fix it to dump the config, thank you.